sample-queries/sample-queries.json

```diff
@@ -1823,6 +1823,17 @@
         "postBody": "{\r\n  \"applicationName\": \"LinkedResource created from Microsoft Graph Explorer\"\r\n}",
         "tip": "This query requires the Tasks.ReadWrite permission. To find a value for the taskListId parameter, you can run: GET https://graph.microsoft.com/beta/me/todo/lists. to find a value for the taskId, you can run: GET https://graph.microsoft.com/beta/me/todo/lists/{taskListId}",
         "skipTest": false
-    }
+    },
+    {
+        "id": "0FB601F7-2170-40DE-88B7-4D7387C7B547",
+        "category": "Identity and Access",
+        "method": "GET",
+        "humanName": "count the users in your tenant",
+        "requestUrl": "/beta/users/$count",
+        "headers": [ { "name": "ConsistencyLevel", "value": "eventual" } ],
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/user-list?view=graph-rest-beta&tabs=http#example-6-get-only-a-count-of-users",
+        "tip": "You are using the advanced query capabilities for Directory Objects, please <a href='https://aka.ms/MsGraphAADSurveyGE' target='_blank'>send us feedback here</a>!",
```

Comment:
@Licantrop0 remove the a href tags and leave the tip message as 
`"tip": "You are using the advanced query capabilities for Directory Objects, please send us feedback here https://aka.ms/MsGraphAADSurveyGE"`
as @thewahome suggested.